distribution/docs/src/main/resources/_contents/_security-framework/security-rest-intro-contents.adoc

```diff
@@ -0,0 +1,1255 @@
+                                <saml2:AttributeValue xsi:type="xs:string">admin</saml2:AttributeValue>
+                            </saml2:Attribute>
+                        </saml2:AttributeStatement>
+                    </saml2:Assertion>
+                </RequestedSecurityToken>
+                <RequestedAttachedReference>
+                    <ns3:SecurityTokenReference xmlns:wsse11="http://docs.oasis-open.org/wss/oasis-wss-wssecurity-secext-1.1.xsd" wsse11:TokenType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0">
+                        <ns3:KeyIdentifier ValueType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLID">_7437C1A55F19AFF22113672577526132</ns3:KeyIdentifier>
+                    </ns3:SecurityTokenReference>
+                </RequestedAttachedReference>
+                <RequestedUnattachedReference>
+                    <ns3:SecurityTokenReference xmlns:wsse11="http://docs.oasis-open.org/wss/oasis-wss-wssecurity-secext-1.1.xsd" wsse11:TokenType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0">
+                        <ns3:KeyIdentifier ValueType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLID">_7437C1A55F19AFF22113672577526132</ns3:KeyIdentifier>
+                    </ns3:SecurityTokenReference>
+                </RequestedUnattachedReference>
+                <wsp:AppliesTo xmlns:wsp="http://schemas.xmlsoap.org/ws/2004/09/policy" xmlns:wst="http://docs.oasis-open.org/ws-sx/ws-trust/200512">
+                    <wsa:EndpointReference xmlns:wsa="http://www.w3.org/2005/08/addressing">
+                        <wsa:Address>https://server:8993/services/QueryService</wsa:Address>
+                    </wsa:EndpointReference>
+                </wsp:AppliesTo>
+                <Lifetime>
+                    <ns2:Created>2013-04-29T17:49:12.620Z</ns2:Created>
+                    <ns2:Expires>2013-04-29T18:19:12.620Z</ns2:Expires>
+                </Lifetime>
+            </RequestSecurityTokenResponse>
+        </RequestSecurityTokenResponseCollection>
+    </soap:Body>
+</soap:Envelope>
+----
+
+===== X.509 PublicKey SAML Security Token Request/Response
+
+In order to obtain a SAML assertion to use in secure communication to ${branding}, a `RequestSecurityToken` (RST) request has to be made to the STS.
+
+An endpoint's policy will specify the type of security token needed.
+Most of the endpoints that have been used with ${branding} require a SAML v2.0 assertion with a required KeyType of http://docs.oasis-open.org/ws-sx/ws-trust/200512/PublicKey.
+This means that the SAML assertion provided by the client to a ${branding} endpoint must contain a SubjectConfirmation block with a type of "holder-of-key" containing the client's public key.
+This is used to prove that the client can possess the SAML assertion returned by the STS.
+
+====== Request
+
+*Explanation*
```

Comment:
We should follow consistent formatting on `Explanation`. It was on a separate line here: https://github.com/codice/ddf/pull/1244/files#diff-347609d2d90dc52edb78eadf411c6d73R732